Assets/XML/Text/Civilopedia.xml

```diff
@@ -14,8 +14,8 @@
zzatasi, cominci� a diventare obsoleta). L'Impero ottomano aveva i caratteri dello stato dinastico pre-moderno, comprendente diverse etnie (e difatti la burocrazia, vitale per i suoi successi, era composta principalmente da greci). Militarmente, la potenza degli Ottomani si doveva all'immediata adozione della polvere da sparo e alla sua potente cavalleria. L'esercito vantava anche un corpo speciale di soldati schiavi noti come "giannizzeri", i veri artefici dell'espansione ottomana, e rimase competitivo fino alla fine del XVIII secolo, quando rimase tecnologicamente indietro rispetto a quelli dell'Europa occidentale.[PARAGRAPH:2]Gli Ottomani erano noti per la loro tolleranza religiosa. L'Impero era uno stato islamico e il sistema legale era dominato dalla legge islamica, ma i non musulmani erano generalmente trattati bene e avevano libert&#224; di religione, tanto che l'Impero ottomano divenne un sicuro rifugio per gli ebrei di tutta Europa. Dal punto di vista economico, l'Impero era poco avanzato. Buona parte dei traffici della regione era in mano ai mercanti greci o italiani, cosa che fren� lo sviluppo dell'industria turca. Quando, in seguito ai viaggi d'esplorazione europei, si tracciarono nuove rotte per raggiungere India e Cina che non passavano per il Vicinio Oriente, l'Impero ottomano inizi� a vivere un lento declino commerciale.[PARAGRAPH:2]All'inizio del XIX secolo l'Impero ottomano si estendeva ancora su un territorio vastissimo, ma era divenuto stagnante e corrotto ed era considerato come il grande "malato d'Europa". Le potenze occidentali, la cui forza commerciale e militare compirono un grosso passo in avanti con la Rivoluzione Industriale, iniziarono a riconquistare territori agli Ottomani. Gli Inglesi e i Francesi li costrinsero a trattati commerciali umilianti e l'Impero fu costretto a contrarre enormi prestiti per pagare i propri debiti: l'inizio di un circolo vizioso che non port� che a debiti sempre maggiori per l'Impero.[PARAGRAPH:2]Anche se furono intraprese delle serie riforme per modernizzare l'impero, i costumi e le tradizioni impedirono ai riformatori di agire con la necessaria celerit&#224;. Inoltre la crescente ondata di nazionalismo cominci� a dilaniare lo stato: Serbi, Bulgari, Rumeni e Greci si dichiararono tutti indipendenti e fondarono i rispettivi stati. All'epoca dello scoppio della Prima Guerra Mondiale, l'Impero ottomano aveva perduto quasi tutti i suoi possedimenti in Europa. Nel disperato tentativo di conservare il poco rimasto, gli Ottomani si allearono con le potenze centrali (Germania e Austria), solo per essere sconfitti dalla rivolta araba del Vicino Oriente guidata dai Britannici. Al termine della guerra i vincitori si ripartirono le spoglie dell'Impero ottomano alla conferenza di pace di Parigi, anche se la Rivoluzione Turca imped� la spartizione anche dell'Anatolia. L'Impero ottomano conobbe la fine nel 1922 con la fondazione dello stato turco e la deposizione dell'ultimo sultano, Mehmet VI.</Italian>
```

Comment:
I see a lot of � in the Ottoman Empire text

Reply:
IIRC that was copied from the original Warlords text. I have no idea why it got corrupted. Anyway, I recopied the original Warlords text and it should be fixed now.